packages/light-client/README.md

```diff
@@ -18,13 +18,13 @@ The evolution of light clients is emblematic of the broader trajectory of Ethere
 
 ## Requirements for Running a Light-Client
 
-Access to an beacon node that supports the light client specification is necessary. The client must support the following routes from the [consensus API spec](https://github.com/ethereum/consensus-specs/tree/dev):
+Access to an beacon node that supports the light client specification is necessary. The client must support the following routes from the [consensus API spec](https://github.com/ethereum/beacon-APIs/tree/v2.5.0/apis/beacon/light_client):
 
-- `/eth/v1/beacon/light_client/updates`
-- `/eth/v1/beacon/light_client/optimistic_update`
-- `/eth/v1/beacon/light_client/finality_update`
-- `/eth/v1/beacon/light_client/bootstrap/{block_root}`
-- `/eth/v0/beacon/light_client/committee_root`
+- `GET /eth/v1/beacon/light_client/updates`
```

Comment:
How do you feel about making these links to the api docs?

```suggestion
- [`GET /eth/v1/beacon/light_client/updates`](https://ethereum.github.io/beacon-APIs/#/Beacon/getLightClientUpdatesByRange)
```

Reply:
great idea, we should also drop the v0, as far as I can see it's not required